script/Ciel/Day 10/10_01_CIEL10_1_2.txt

```diff
@@ -48,7 +48,7 @@ Could you give my regards to Kohaku-san and Akiha?"
 [sha:05771cd36620c8ed7f806bb6749fbdd167fdfbae]{
 -- Page 64, Offset 27241.
 -- 翡翠にそう声をかけて先を急ぐ。
-After making my excuses to Hisui, I rush out the door.
+After making my excuses to Hisui, I rush out the door.//Not really an excuse more like an apology? Like "After giving my apologies to Hisui, I rush out the door"?
```

Comment:
The more literal tl would be 'After saying that to Hisui, I rush out the door.' I'd prefer to revert back in that direction if we were to make a change.

Reply:
Yeah I think I used excuses since he's _excusing_ himself from the premises, and I don't like plain "I say that and then X" sentence structure in EN.